test/publishing_api_v2_test.rb

```diff
@@ -1232,4 +1232,24 @@ def content_item_for_content_id(content_id, attrs = {})
       end
     end
   end
+
+  describe "content ID validation" do
+    [:get_content, :get_links, :get_linked_items, :discard_draft].each do |method|
```

Comment:
I think this was only [introduced in Ruby 2.0](http://dev.venntro.com/2013/03/ruby-2-0-0-in-detail/#i-and-i-symbol-array-literal) and we need to support down to 1.9.3 with `gds-api-adapters`, as far as I know.